### ISSUE TITLE
createCredentials help

### ISSUE THREAD

**Issue by danyal** [crypto-browserify/crypto-browserify#147]
Could someone point me in the right direction of implementing createCredentials? Not much experience on this front but would like to try to dig into it.

**Comment by calvinmetcalf** [comment]
Ok so the end result you want to create is a module that implimentats it
that we require here.

Step one is to document the api and create some examples of test vectors or
figure out how to validate the out put of what you are making.  Likely the
hard part here is finding documentation on how the SPKAC stuff is formated.

If you do this first then if you don't do anything more than somebody else
can pick up where you left off easily.

For the actual public key stuff you'll want to use asn1.js for the encoding
and decoding of the stuff. randombytes for generating random data and
browserify-sign if you need to create signatures.

**Comment by fanatid** [comment]
`createCredentials` in node `crypto` marked as deprecated (starting from [node v0.12](https://nodejs.org/docs/latest-v0.12.x/api/crypto.html#crypto_crypto_createcredentials_details))
@calvinmetcalf should it be removed from [todo](https://github.com/crypto-browserify/crypto-browserify/tree/eea13e59d89978d0d8fb5b237bcead3dfd9add5e#todo)?

**Comment by ljharb** [comment]
Deprecated is irrelevant; if it exists, it should be added.

**Comment by coltongit** [comment]
Agreed. If someone is willing to make this as a module, then be my guest. If you want to add this as a module, please do so as per the readme, @danyal.
